service/spi/src/test/java/org/kaazing/gateway/service/messaging/collections/MemoryCollectionsFactoryTest.java

```diff
@@ -251,7 +272,7 @@ public void expiringMap() throws Exception {
         map.putIfAbsent("two", "2", 10000, MILLISECONDS);
         map.putIfAbsent("three", "3", 2, MILLISECONDS);
 
-        sleep(5);
+        sleep(10);
```

Comment:
I would not expect this test to require a change.  Moreover, I think the `assertEquals("2", map.putIfAbsent("two", "20", 1000, MILLISECONDS));` would fail with this change